plugins/module_utils/system_access_users_utils.py

```diff
@@ -39,70 +39,114 @@
     xml_utils,
     opnsense_utils,
 )
+
 from ansible_collections.puzzle.opnsense.plugins.module_utils.config_utils import (
     OPNsenseModuleConfig,
 )
-from ansible_collections.puzzle.opnsense.plugins.module_utils.enum_utils import ListEnum
 
 
-class OPNSenseGroupNotFoundError(Exception):
+class OPNSenseCryptReturnError(Exception):
     """
-    Exception raised when an OPNsense group is not found.
+    Exception raised when the return value of the instance is not what is expected
     """
 
 
-class OPNSenseNotValidBase64APIKeyError(Exception):
+class OPNSenseGroupNotFoundError(Exception):
```

Comment:
```suggestion
class OPNsenseGroupNotFoundError(Exception):
```

Reply:
Thanks for the suggestion, fixed in c5b1588.